<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>client_window</class>
 <widget class="QMainWindow" name="client_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>client_window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>921</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
  background: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1,
                              stop: 0 #025bad, stop: 0.1#025bad,
                              stop: 0.5#51c6f0, stop: 0.9#025bad,

                              stop: 1.0 #025bad);
  border: 2px solid #8f8f91;
  border-radius: 6px;
  padding: 6px;
}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_crud_c">
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(0,0,0,0);
color: rgb(255, 255, 255);
border-color:rgba(0,0,0,0);  font-size: 17px;</string>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>260</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(0,0,0,0);
color: rgb(255, 255, 255);
border-color:rgba(0,0,0,0);  font-size: 17px;</string>
      </property>
      <property name="text">
       <string>Sexe</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>180</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(0,0,0,0);
color: rgb(255, 255, 255);
border-color:rgba(0,0,0,0);  font-size: 17px;</string>
      </property>
      <property name="text">
       <string>Telephone</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>140</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(0,0,0,0);
color: rgb(255, 255, 255);
border-color:rgba(0,0,0,0);  font-size: 17px;</string>
      </property>
      <property name="text">
       <string>Adress</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="client_cin">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>20</y>
        <width>110</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: #FFFFFF;
    border: 1px solid #BBBBBB;
    border-radius: 5px;
    padding: 3px;
    font-size: 15px;
    font-family: &quot;Arial&quot;;
}

QLineEdit:focus {
    border: 2px solid #0000FF;
    background-color: #F0F0F0;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="client_tel">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>180</y>
        <width>110</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: #FFFFFF;
    border: 1px solid #BBBBBB;
    border-radius: 5px;
    padding: 3px;
    font-size: 15px;
    font-family: &quot;Arial&quot;;
}

QLineEdit:focus {
    border: 2px solid #0000FF;
    background-color: #F0F0F0;
}
</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="client_adress">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>140</y>
        <width>110</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: #FFFFFF;
    border: 1px solid #BBBBBB;
    border-radius: 5px;
    padding: 3px;
    font-size: 15px;
    font-family: &quot;Arial&quot;;
}

QLineEdit:focus {
    border: 2px solid #0000FF;
    background-color: #F0F0F0;
}
</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="client_email">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>300</y>
        <width>110</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: #FFFFFF;
    border: 1px solid #BBBBBB;
    border-radius: 5px;
    padding: 3px;
    font-size: 15px;
    font-family: &quot;Arial&quot;;
}

QLineEdit:focus {
    border: 2px solid #0000FF;
    background-color: #F0F0F0;
}
</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="client_prenom">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>60</y>
        <width>110</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: #FFFFFF;
    border: 1px solid #BBBBBB;
    border-radius: 5px;
    padding: 3px;
    font-size: 15px;
    font-family: &quot;Arial&quot;;
}

QLineEdit:focus {
    border: 2px solid #0000FF;
    background-color: #F0F0F0;
}
</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>300</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(0,0,0,0);
color: rgb(255, 255, 255);
border-color:rgba(0,0,0,0);  font-size: 17px;</string>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(0,0,0,0);
color: rgb(255, 255, 255);
border-color:rgba(0,0,0,0);  font-size: 17px;</string>
      </property>
      <property name="text">
       <string>Prenom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(0,0,0,0);
color: rgb(255, 255, 255);
border-color:rgba(0,0,0,0);  font-size: 17px;</string>
      </property>
      <property name="text">
       <string>Cin</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>220</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(0,0,0,0);  font-size: 17px;
color: rgb(255, 255, 255);
border-color:rgba(0,0,0,0);</string>
      </property>
      <property name="text">
       <string>Date </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="client_nom">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>100</y>
        <width>110</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: #FFFFFF;
    border: 1px solid #BBBBBB;
    border-radius: 5px;
    padding: 3px;
    font-size: 15px;
    font-family: &quot;Arial&quot;;
}

QLineEdit:focus {
    border: 2px solid #0000FF;
    background-color: #F0F0F0;
}
</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="client_rech">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>490</y>
        <width>110</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: #FFFFFF;
    border: 1px solid #BBBBBB;
    border-radius: 5px;
    padding: 3px;
    font-size: 10px;
    font-family: &quot;Arial&quot;;
}

QLineEdit:focus {
    border: 2px solid #0000FF;
    background-color: #F0F0F0;
}</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="client_date">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>220</y>
        <width>110</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QDateEdit {
    border: 2px solid gray;
    border-radius: 5px;
    background-color: white;
    padding: 2px 4px;
    color: black;
}

QDateEdit::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;
    border-left-width: 1px;
    border-left-color: gray;
    border-left-style: solid;
    border-top-right-radius: 3px;
    border-bottom-right-radius: 3px;
}

QDateEdit::down-arrow {
    image: url(:/icons/down-arrow.png);
    width: 10px;
    height: 10px;
}</string>
      </property>
     </widget>
     <widget class="QTableView" name="client_table">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>70</y>
        <width>571</width>
        <height>321</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableView {
    background-color: white;
    border: 1px solid black;
    font-size: 14px;
    selection-background-color: blue; /* set selection background color to blue */
}

QHeaderView::section {
    background-color: white;
    color: black;
    font-weight: bold;
    padding: 5px;
    text-align: left;
    height: 40px;
}

QTableView::item {
    border-bottom: 1px solid black;
    padding: 5px;
}</string>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="recherche_affiche_6">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>430</y>
        <width>140</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: white;
    color: black;
    border: 2px solid black;
    border-radius: 10px;
    padding: 5px;
    font-size: 14px;
}

QPushButton:hover {
    background-color: #737370;
    color: white;
    border: 2px solid white;
    transform: scale(1.1, 1.1);
    transition: all 0.5s ease;
}</string>
      </property>
      <property name="text">
       <string>afficher statestic</string>
      </property>
     </widget>
     <widget class="QComboBox" name="client_tri">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>430</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
    border: 1px solid gray;
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
}

QComboBox:editable {
    background: white;
}

QComboBox:!editable, QComboBox::drop-down:editable {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                 stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
}

/* QComboBox gets the &quot;on&quot; state when the popup is open */
QComboBox:!editable:on, QComboBox::drop-down:editable:on {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #D3D3D3, stop: 0.4 #D8D8D8,
                                stop: 0.5 #DDDDDD, stop: 1.0 #E1E1E1);
}

QComboBox:on { /* shift the text when the popup opens */
    padding-top: 3px;
    padding-left: 4px;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;

    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid; /* just a single line */
    border-top-right-radius: 3px; /* same radius as the QComboBox */
    border-bottom-right-radius: 3px;
}

QComboBox::down-arrow {
    image: url(/usr/share/icons/crystalsvg/16x16/actions/1downarrow.png);
}

QComboBox::down-arrow:on { /* shift the arrow when popup is open */
    top: 1px;
    left: 1px;
}</string>
      </property>
      <item>
       <property name="text">
        <string>normal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>tri_croissant</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>tri_decroissant</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="client_sexe">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>260</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
    border: 1px solid gray;
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
}

QComboBox:editable {
    background: white;
}

QComboBox:!editable, QComboBox::drop-down:editable {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                 stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
}

/* QComboBox gets the &quot;on&quot; state when the popup is open */
QComboBox:!editable:on, QComboBox::drop-down:editable:on {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #D3D3D3, stop: 0.4 #D8D8D8,
                                stop: 0.5 #DDDDDD, stop: 1.0 #E1E1E1);
}

QComboBox:on { /* shift the text when the popup opens */
    padding-top: 3px;
    padding-left: 4px;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;

    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid; /* just a single line */
    border-top-right-radius: 3px; /* same radius as the QComboBox */
    border-bottom-right-radius: 3px;
}

QComboBox::down-arrow {
    image: url(/usr/share/icons/crystalsvg/16x16/actions/1downarrow.png);
}

QComboBox::down-arrow:on { /* shift the arrow when popup is open */
    top: 1px;
    left: 1px;
}</string>
      </property>
      <item>
       <property name="text">
        <string>Male</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Female</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="client_recherche">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>480</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: white;
    color: black;
    border: 2px solid black;
    border-radius: 10px;
    padding: 5px;
    font-size: 14px;
}

QPushButton:hover {
    background-color: #737370;
    color: white;
    border: 2px solid white;
    transform: scale(1.1, 1.1);
    transition: all 0.5s ease;
}</string>
      </property>
      <property name="text">
       <string>Client a rechercher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="client_modifier">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>420</y>
        <width>140</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #04c8d6, stop: 1 #04c8d6);
    border: 2px solid #000000;
    border-radius: 10px;
    padding: 10px;
	
	
    color: white;
    font-size: 14px;
    font-weight: bold;
    text-align: center;
}

QPushButton:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #50faf7, stop: 1 #50faf7);
    cursor: pointer;
    box-shadow: 0px 0px 5px #004040;
}

QPushButton:pressed {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #004080, stop: 1 #002040);
}</string>
      </property>
      <property name="text">
       <string>modifier client</string>
      </property>
     </widget>
     <widget class="QPushButton" name="client_supprimer">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>480</y>
        <width>140</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #04c8d6, stop: 1 #04c8d6);
    border: 2px solid #000000;
    border-radius: 10px;
    padding: 10px;
	
	
    color: white;
    font-size: 14px;
    font-weight: bold;
    text-align: center;
}

QPushButton:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #50faf7, stop: 1 #50faf7);
    cursor: pointer;
    box-shadow: 0px 0px 5px #004040;
}

QPushButton:pressed {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #004080, stop: 1 #002040);
}</string>
      </property>
      <property name="text">
       <string>supprimer client</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>20</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: white;
    color: black;
    border: 2px solid black;a
    border-radius: 10px;
    padding: 5px;
    font-size: 14px;
}

QPushButton:hover {
    background-color: #737370;
    color: white;
    border: 2px solid white;
    transform: scale(1.1, 1.1);
    transition: all 0.5s ease;
}</string>
      </property>
      <property name="text">
       <string>return</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>430</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: white;
    color: black;
    border: 2px solid black;
    border-radius: 10px;
    padding: 5px;
    font-size: 14px;
}

QPushButton:hover {
    background-color: #737370;
    color: white;
    border: 2px solid white;
    transform: scale(1.1, 1.1);
    transition: all 0.5s ease;
}</string>
      </property>
      <property name="text">
       <string>afficher client</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ajouter_client">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>360</y>
        <width>140</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #04c8d6, stop: 1 #04c8d6);
    border: 2px solid #000000;
    border-radius: 10px;
    padding: 10px;
	
	
    color: white;
    font-size: 14px;
    font-weight: bold;
    text-align: center;
}

QPushButton:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #50faf7, stop: 1 #50faf7);
    cursor: pointer;
    box-shadow: 0px 0px 5px #004040;
}

QPushButton:pressed {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #004080, stop: 1 #002040);
}</string>
      </property>
      <property name="text">
       <string>ajouter client</string>
      </property>
     </widget>
     <widget class="QPushButton" name="client_convert_to_pdf">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>430</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: white;
    color: black;
    border: 2px solid black;
    border-radius: 10px;
    padding: 5px;
    font-size: 14px;
}

QPushButton:hover {
    background-color: #737370;
    color: white;
    border: 2px solid white;
    transform: scale(1.1, 1.1);
    transition: all 0.5s ease;
}</string>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="afficher_stats_client">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>440</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: white;
    color: black;
    border: 2px solid black;
    border-radius: 10px;
    padding: 5px;
    font-size: 14px;
}

QPushButton:hover {
    background-color: #737370;
    color: white;
    border: 2px solid white;
    transform: scale(1.1, 1.1);
    transition: all 0.5s ease;
}</string>
      </property>
      <property name="text">
       <string>afficher stats</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>754</x>
        <y>30</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: white;
    color: black;
    border: 2px solid black;a
    border-radius: 10px;
    padding: 5px;
    font-size: 14px;
}

QPushButton:hover {
    background-color: #737370;
    color: white;
    border: 2px solid white;
    transform: scale(1.1, 1.1);
    transition: all 0.5s ease;
}</string>
      </property>
      <property name="text">
       <string>return</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="client_stats">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>70</y>
        <width>511</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>942</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
